.azure/pipelines/ci.yml

```diff
@@ -332,6 +332,8 @@ stages:
             arguments: $(Build.SourcesDirectory)/NuGet.config $Token
           env:
             Token: $(dn-bot-dnceng-artifact-feeds-rw)
+      - script: df -h
+        displayName: Disk utilization (before)
```

Comment:
Suggest doing this in &hellip;/jobs/default-build.yml instead with new steps excluding Windows i.e. preceded by
```yml
- ${{ if ne(parameters.agentOs, 'Windows') }}:
```

And, as @MattGal suggested, let's do this for realz.

Reply:
Updatd